services/web/pkg/config/config.go

```diff
@@ -30,7 +31,9 @@ type Config struct {
 
 // Asset defines the available asset configuration.
 type Asset struct {
-	Path string `yaml:"path" env:"WEB_ASSET_PATH" desc:"Serve ownCloud Web assets from a path on the filesystem instead of the builtin assets."`
+	DeprecatedPath string `yaml:"path" env:"WEB_ASSET_PATH" desc:"Serve ownCloud Web assets from a path on the filesystem instead of the builtin assets." deprecationVersion:"5.1.0" removalVersion:"6.0.0" deprecationInfo:"The WEB_ASSET_PATH is deprecated and will be removed in the future." deprecationReplacement:"Use WEB_ASSET_CORE_PATH instead."`
```

Comment:
Just a note, there will be changes in envvars so it will pass [check-env-var-annotations](https://github.com/owncloud/ocis/pull/8469)

Reply:
thanks, is there currently any action item here?

Reply:
yes, you **MUST** to add to EACH new envvar ` introductionVersion:"5.1"`
See the reason at: https://github.com/owncloud/ocis/pull/8374#pullrequestreview-1916457031 and 

https://github.com/owncloud/ocis/pull/8454/files#diff-847473017ffefbf34c2b2c27d4f3a54effb5a58983b3dd83a7f647bc34ccaa3c